concordium-consensus/src/Concordium/KonsensusV1/TreeState/Implementation.hs

```diff
@@ -229,8 +229,9 @@ data SkovData (pv :: ProtocolVersion) = SkovData
       _skovPendingBlocks :: !PendingBlocks,
       -- |Pointer to the last finalized block.
       _lastFinalized :: !(BlockPointer pv),
-      -- |Certified block that justifies the last finalized block being finalized.
-      _finalizingCertifiedBlock :: !(Option (CertifiedBlock pv)),
+      -- |A finalization entry that finalizes the last finalized block, unless that is the
```

Comment:
the `_branches` above. Isn't it an invariant that (certified blocks) in the branches must be present also in the lmdb database?

Reply:
This is not really an invariant of `_branches`. I'll add a comment on the block table.